standard/docs/en/support/governance.md

```diff
@@ -200,5 +200,15 @@ Anyone who is a current or potential publisher or user of the standard can be co
 
 "The principle of consensus has its origins in the desire to achieve the general acceptance and application of a Standard within its intended sphere of influence. This entails trying to ensure that the interests of all those likely to be affected by it are taken into account, and that individual concerns are carefully and fairly balanced against the wider public interest." [BSI, 2012](http://www.bsigroup.com/Documents/about-bsi/NSB/BSI-pocket-guide-to-standards-development-UK-EN.pdf) 
 
-   
+## Versioning of extensions
+
+Extensions are governed and versioned independently of the standard, with the exception of core extensions, where:
+
+* A MAJOR or MINOR release of a core extension will only take place as part of the governance process for a MAJOR or MINOR release of OCDS. 
+
+* Each version of the standard will explicitly declare the version number of each core extension for that version of the standard. 
+
+Changes to core extensions between standard versions may be staged with -alpha, or -beta suffixes to indicate these are under development or proposed for adoption in the next version of OCDS, but that their adoption has not yet been formally agreed. 
```

Comment:
I suggest "with labels (like `-alpha` or `-beta`)" instead of "with -alpha, or -beta suffixes", to allow for `-rc.1` etc.

Reply:
Done in https://github.com/open-contracting/standard/pull/674/commits/ef99fb58b6213da3d9e35b4b87a54b2a78697990